### PR TITLE
Added unix primer / guide

### PR DESCRIPTION
**Pull request by stuwilkins**
Attn @cmazzoli @ambarb, this is work in progress but acts as a primer for files / permissions etc. Comments and contributions very welcome! 

The compiled output can be found [here](http://nsls-ii-csx.github.io/)

### CODE REVIEW

**Comment by stuwilkins** [comment]
Added the instructions to install miniconda3 and setup custom environment. 

Attn @ambarb @cmazzoli

**Comment by stuwilkins** [comment]
Attn @danielballan any comments very welcome. 

@ambarb , @cmazzoli  take a look at:

http://nsls-ii-csx.github.io/guides/conda_env.html

**Comment by danielballan** [comment]
This is good. I wish this had been here when I was a new employee. I see no problems other than the "personal miniconda vs. root conda" thing which we discussed -- I agree it's fine this way for now.

I will borrow some content for the main facility-wide docs. Thanks for this effort.

File: guides/vim-cheatsheet.rst [hunk 1]
```diff
@@ -0,0 +1,444 @@
+Vim Commands Cheat Sheet
```
Comment (tacaswell) [inline guides/vim-cheatsheet.rst:1]: This could be a lot shorter:

`:q!`
`emacs`

:smiling_imp:

Comment (stuwilkins) [inline guides/vim-cheatsheet.rst:1]: Very funny … sphinx is still compiling the emacs cheat sheet.. more news soon.  ;-)

Comment (danielballan) [inline guides/vim-cheatsheet.rst:1]: I read a good one today: "I've been using vim since 1996, but only because I can't figure out how to exit."

**Pull request merged**